libraries/cmake/source/librdkafka/CMakeLists.txt

```diff
@@ -78,6 +79,11 @@ function(generateLibrdkafka)
     "${library_root}/lz4hc.c"
     "${library_root}/rddl.c"
     "${library_root}/rdkafka_plugin.c"
+    "${library_root}/rdkafka_sasl_win32.c"
```

Comment:
This should be included in the list only if the platform is Windows.

Reply:
implemented with the PLATFORM checks :)